worker/include/RTC/TransportTuple.hpp

```diff
@@ -125,6 +130,28 @@ namespace RTC
 				return this->tcpConnection->GetSentBytes();
 		}
 
+	private:
+		void GenerateHash()
```

Comment:
Name doesn't reflect what is happening clearly:
```suggestion
		void UpdateOwnHash()
```

Reply:
Why "update"? It's not updating the hash but generating it for first time (and it will never change).

Reply:
`GenerateHash` has a name that implies that the hash will be returning. I found it confusing that it reads internal state and updated object's property.

Reply:
What about `SetHash()`?

Reply:
I expect `SetHash()` to take an argument with the hash, but it is a better name than `GenerateHash()`

Reply:
If `SetHash()` signature doesn't contain any argument it's clear that it sets something by its own :)
I'll change this.